src/main/twirl/views/shared/_coverageInformation.scala.html

```diff
@@ -1,41 +1,39 @@
 <p>
-    The information you provide must relate to 'qualifying contracts'. A qualifying contract is a contract which
-    satisfies all of the following:
+    The information you provide must relate to 'qualifying contracts'. </p>
+<p> A qualifying contract is a contract which satisfies all of the following:
 </p>
 <ul class="list list-bullet">
-    <li>It is between two (or more) businesses</li>
+    <li>It is between 2 (or more) businesses</li>
 
-    <li>It has a significant connection with the United Kingdom (as explained below).</li>
+    <li>It has a significant connection with the UK (as explained below)</li>
 
-    <li>It is for goods, services or intangible property, including intellectual property.</li>
+    <li>It is for goods, services or intangible property, including intellectual property</li>
 
-    <li>It is not for <a rel="external" target="_blank" href="http://www.legislation.gov.uk/ukpga/2015/26/section/2/enacted">financial services</a>.</li>
+    <li>It is not for <a rel="external" target="_blank" href="http://www.legislation.gov.uk/ukpga/2015/26/section/2/enacted">financial services</a></li>
 </ul>
 
 <h3 class="heading-small">Significant Connection to the UK</h3>
 
 <p>
-    If a UK law (i.e. the law of England and Wales, Scotland or Northern Ireland) applies to a contract without the parties
```

Comment:
Everything from here below will take longer to sign off and therefore should be in a separate PR